tests/CMakeLists.txt

```diff
@@ -20,12 +20,14 @@ function(add_cosma_mpi_test test_name num_procs)
              COMMAND ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} ${num_procs}
                      ${MPIEXEC_PREFLAGS} ./${tgt_} ${MPIEXEC_POSTFLAGS})
     add_dependencies(tests ${tgt_})
+    install(TARGETS ${tgt_} RUNTIME DESTINATION "${CMAKE_INSTALL_PREFIX}/bin")
```

Comment:
`${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR}` is slightly more portable : https://cmake.org/cmake/help/v3.0/module/GNUInstallDirs.html